bin/pyang

```diff
@@ -299,6 +304,14 @@ Validates the YANG module in <filename> (or stdin), and all its dependencies."""
                     "module '%s' specified in hello not found.\n" % emarg)
                 sys.exit(1)
             modules.append(mod)
+    elif o.schema_from_path:
+        for module_name in ctx.revs.keys():
+            module = ctx.search_module(0, module_name)
```

Comment:
You're passing '0' as 'pos'.  I can see this is also done in the 'hello' case, but it is actually not correct, b/c error.py expects a Position object.  I think we should use Position('<cmdline>') or something in both these cases.

Reply:
Latest commit has changed `ctx.search_module(0` in `bin/pyang` to `error.Position('')`.